### ISSUE TITLE
Calibrating Offset Hardcoded

### ISSUE THREAD

**Issue by arihantdaga** [mastrogippo/ATM90E3x-open-PC-interface#1]
Thank you very much for this project. I could not find any good programming implementation for this IC anywhere else. I went through the code. Just one thing where i have doubt is why did you hardcoded the Offset Calibration. I see the commented code - 
``` 
WriteReg(0x63, OffsetCal(0xD9));

...

```
Is there a reason, it should be hardcoded?

**Comment by mastrogippo** [comment]
It's not hardcoded, OffsetCal is a function that, in that line for example, calculates the Offset to apply to the register 0x63 based on the value read from register 0xD9!

**Comment by arihantdaga** [comment]
Hi i am sorry, i wasnt clear. I am talking about this block here - 
https://github.com/mastrogippo/ATM90E3x-open-PC-interface/blob/41863a428640ef4a343915b8d6a80a8e808be78e/ATM90E36/Form1.cs#L372-L390


You are not using OffsetCal function instead using this hardcoded value -  `WriteReg(0x63, 0xFC60); `

**Comment by mastrogippo** [comment]
You're right - I think it was a temporary test, in my local copy the other part is commented. I'll fix that, in the meantime you can comment the fixed values and uncomment the functions :)
